app/src/main/java/org/wikipedia/WikipediaApp.kt

```diff
@@ -137,6 +133,9 @@ class WikipediaApp : Application() {
     override fun onCreate() {
         super.onCreate()
 
+        // Set the instance after the Application is fully initialized.
+        instance = this
```

Comment:
Why is this safer? (What is unsafe about setting `instance = this` in the init block?)
Can you say more about why this would fix the crash?

Reply:
The `init` block is the first to be executed when the class is instantiated; the `onCreate` block is about the Android's lifecycle. By considering the crashes are related to the `Prefs` usage, which uses, `WikipediaApp.instance.resource` to get the resource from the app, I think the fix of this PR is a possible way to fix this issue.